komodo/check_up_to_date_pypi.py

```diff
@@ -96,7 +96,11 @@ def get_upgrade_proposals_from_pypi(
 ) -> dict:
     pypi_packages = get_pypi_packages(releases, repository)
     pypi_responses = get_pypi_info(pypi_packages)
+    pypi_responses = get_pypi_info(pypi_packages)
 
+    upgrade_proposals_from_pypi = {}
+    for package_name, _response in pypi_responses:
+        komodo_version = get_version.parse(strip_version(releases[package_name]))
```

Comment:
Duplicated code?

Reply:
there was a merge conflict, so it kept both changes...

Reply:
I think I got all of the duplicates